servers/rendering/rasterizer_rd/shaders/scene_high_end.glsl

```diff
@@ -2685,7 +2685,7 @@ FRAGMENT_SHADER_CODE
 	frag_color = vec4(albedo, alpha);
 #else
 	frag_color = vec4(emission + ambient_light + diffuse_light + specular_light, alpha);
-	//frag_color = vec4(1.0);
+	//frag_color = vec4(1.0);;;
```

Comment:
This is the most important change in this PR.

Reply:
Oh actually this happens because if you change only a .inc, the glsl its included for does not detect the dependency and does not compile, so sometimes I add one of these to make it compile. We should fix this but I am not entirely familiar with how to do it.